reference/src/introduction.md

```diff
@@ -8,7 +8,7 @@
 * *niche*
 * *layout*
 * *tag*
-* *rvalue*
-* *lvalue*
+* *place* (or *lvalue* in C/C++ speak)
+* *rvalue* (maybe we can come up with a Rust term for this as well?)
```

Comment:
These are called "values of the expression" in C, "prvalue" in C++.

Reply:
(and we should just call them "values")

Reply:
Isn't there a term to distinguish expressions like "3+2" from expressions like "x", where the latter can be used as lvalues but the former cannot?

Reply:
Yes, the latter is a (Rust) place expression, (C) lvalue expression, or (C++) glvalue expression.